src/main/java/github/scarsz/discordsrv/util/DebugUtil.java

```diff
@@ -142,6 +143,17 @@ public static String run(String requester, int aesBits) {
         return uploadReport(files, aesBits, requester);
     }
 
+    private static String listGuilds() {
+        if (DiscordUtil.getJda() == null) return "\n   null JDA";
+        String list = "";
+        for (Guild server : DiscordUtil.getJda().getGuilds()) {
+            list += "\n   " + server + "  :  [";
```

Comment:
```suggestion
            list += "\n   " + server + ":  [";
```
Spacing looks a bit weird